src/scooters.ts

```diff
@@ -113,13 +114,13 @@ async function getScooters(
         operatorsWhitelist.map((operator) => {
             switch (operator) {
                 case Operator.VOI:
-                    return getVoiScooters()
+                    return getCachedScooters(operator, () => getVoiScooters())
                 case Operator.TIER:
-                    return getTierScooters(lat, lon, range)
+                    return getCachedScooters(operator, () => getTierScooters(lat, lon, range))
```

Comment:
Kanskje droppe bruken av `getCachedScooters` her,  i staden for å ha spesial-case for Tier inni `getCachedScooters`?